docs/src/main/sphinx/connector/object-storage-file-formats.md

```diff
@@ -88,7 +88,12 @@ with Parquet files performed by supported object storage connectors:
         bloom filters by default. The equivalent catalog session property is
         ``parquet_use_bloom_filter``.
       - ``true``
+    * - ``parquet.use-column-index``
+      - Skip reading Parquet pages by using Parquet column indices. The
+        equivalent catalog session property is ``parquet_use_column_index``.
+        Only supported by the Delta Lake and Hive connectors.
+      - ``true``
     * - ``parquet.max-read-block-row-count``
-      - Sets the maximum number of rows read in a batch.
+      - Sets the maximum number of rows read in a batch. The equivalent catalog
+        session property is ``parquet_max_read_block_row_count``.
```

Comment:
This markdown is referred from Hudi connector, but the connector doesn't support `parquet_max_read_block_row_count` session property.

Reply:
Updated the docs .. arguably all these inconsistencies across the connectors for the usage of the same reader library are a bit painful and it would be good to clean that up in the code and make it more consistent.